src/main/cpp/subsystems/SmartIntake.cpp

```diff
@@ -59,43 +59,28 @@ void SmartIntake::HandleInput(RobotControlData& input){
             }
         }
             break;
-        case 2:
-        {
-            input.intakeInput.runIntakeIn = true;
-            if (input.smartIntakeInput.laser){
-                input.intakeInput.runIntakeIn = false;
-                input.intakeInput.goToPseudoStowPos = true;
-                ++m_IntakeState;
-            }
-            
-        }
-            break;
-        case 3:
-        {
-            input.intakeInput.goToPseudoStowPos = false;
-            if (input.intakeOutput.intakePos == IntakePos::PSEUDO_STOW)
-            {
-                ++m_IntakeState;
-            }
-        }
-            break;
+        
         default:
+            m_SmartIntakeFlag = false;
             break;
         }
     }
     else
     {
         switch (m_IntakeState)
         {
-        case 0:
+        case 2:
         {
-                /* code */
+            input.intakeInput.runIntakeIn = true;
```

Comment:
checking if the laser hit should be in the true case. The if statement should conditionally transfer to the false case